src/sqerl_client.erl

```diff
@@ -241,19 +212,29 @@ drivermod() ->
             log_and_error({invalid_application_config, sqerl, db_driver_mod, Error})
     end.
 
+%% @doc Returns the config, based on configured config_cb MFA. Defaults to
+%% using squerl_config_env:config/0, which will use the application environment.
+-spec config() -> [{atom(), term()}].
+config() ->
+    {M, F, A} = case envy:get(sqerl, config_cb, undefined, any) of
+        undefined ->
+            {sqerl_config_env, config, []};
+        {Mod, Fun, Args} = MFA when is_atom(Mod) andalso
```

Comment:
I think this is probably ok since only devs are going to use the feature, but otherwise a friendlier error message when wrong type is provided would make sense. Agree?